### PR TITLE
chore: introduce app caching v3

### PR DESCRIPTION
**Pull request by IvanJobs**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/17292817

### CODE REVIEW

File: hello-world-in-meeting/README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # How to use this HelloWorld In-meeting app
 
 > Important: Please be advised that access tokens are stored in sessionStorage for you by default. This can make it possible for malicious code in your app (or code pasted into a console on your page) to access APIs at the same privilege level as your client application. Please ensure you only request the minimum necessary scopes from your client application, and perform any sensitive operations from server side code that your client has to authenticate with.
+> App Caching was configured in this sample to reduce the reload time of your app during a meeting. It has limitations and available scopes, and for more details, please check [Enable app caching for your tab app](https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/app-caching-for-your-tab-app).
```
Comment (sffamily) [inline hello-world-in-meeting/README.md:4]: during -> in
It has limitations and available scopes, and for more details, -> To learn about limitations and available scopes

File: hello-world-in-meeting/src/components/Tab.jsx [hunk 2]
```diff
@@ -15,12 +15,32 @@ class Tab extends React.Component {
   //Learn more: https://reactjs.org/docs/react-component.html#componentdidmount
   componentDidMount(){
     app.initialize().then(() => {
+      // Notifies the frame that app initialization is successfully and is ready for user interaction.
```
Comment (sffamily) [inline hello-world-in-meeting/src/components/Tab.jsx:18]: Notify that app initialization is successful and ....

File: hello-world-in-meeting/src/components/Tab.jsx [hunk 3]
```diff
@@ -15,12 +15,32 @@ class Tab extends React.Component {
...
+
       // Get the user context from Teams and set it in the state
       app.getContext().then(async (context) => {
         this.setState({
           meetingId: context.meeting.id,
           userName: context.user.userPrincipalName
         });
+
+        // Enable app caching in meeting's side panel.
+        // App Caching was configured in this sample to reduce the reload time of your app during a meeting.
```
Comment (sffamily) [inline hello-world-in-meeting/src/components/Tab.jsx:29]: during -> in

File: hello-world-in-meeting/src/components/Tab.jsx [hunk 4]
```diff
@@ -15,12 +15,32 @@ class Tab extends React.Component {
...
+      app.notifySuccess();
+
       // Get the user context from Teams and set it in the state
       app.getContext().then(async (context) => {
         this.setState({
           meetingId: context.meeting.id,
           userName: context.user.userPrincipalName
         });
+
+        // Enable app caching in meeting's side panel.
```
Comment (sffamily) [inline hello-world-in-meeting/src/components/Tab.jsx:28]: is it limited to side panel only? how about stage panel?

recommend to be more generic, like "Enable app caching for meeting app" or just "Enable app caching"

File: hello-world-in-meeting/src/components/Tab.jsx [hunk 5]
```diff
@@ -15,12 +15,32 @@ class Tab extends React.Component {
...
+        // App Caching was configured in this sample to reduce the reload time of your app during a meeting. 
+        // It has limitations and available scopes, and for more details, please check https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/app-caching-for-your-tab-app.
```
Comment (sffamily) [inline hello-world-in-meeting/src/components/Tab.jsx:30]: same as above comment for the same text.

**Review by eriolchan — APPROVED** [review]

**Review by sffamily — APPROVED** [review]

**Pull request merged**